metronome/networking/test/src/io/iohk/metronome/networking/RemoteConnectionManagerWithScalanetProviderSpec.scala

```diff
@@ -129,9 +132,40 @@ object RemoteConnectionManagerWithScalanetProviderSpec {
     FramingConfig.buildStandardFrameConfig(1000000, 4).getOrElse(null)
   val testIncomingQueueSize = 20
 
+  implicit val secp256k1Encoder: Encoder[Secp256k1Key] =
+    Encoder.instance(key => Json.fromString(key.key.toHex))
+
+  // Just an example of setting up logging.
+  implicit def tracers[F[_]: Sync, K: io.circe.Encoder, M]
+      : NetworkTracers[F, K, M] = {
+    import io.circe.syntax._
+    import NetworkEvent._
+
+    def peerJson(key: K, address: InetSocketAddress) =
+      JsonObject("key" -> key.asJson, "address" -> address.toString.asJson)
+
+    implicit val hybridLog: HybridLog[NetworkEvent[K]] =
+      HybridLog.instance[NetworkEvent[K]](
+        level = _ => HybridLogObject.Level.Debug,
```

Comment:
Just to make sure if I understand correctly, If I would like to have differnt log levels for different event I would need to patter match them here for attach correct log levels to them ?

Reply:
Yeah, you can issue a warning/error for certain types if you want to.